src/scene/graph-node.js

```diff
@@ -1155,13 +1156,7 @@ class GraphNode extends EventHandler {
      * this.entity.addChild(e);
      */
     addChild(node) {
-        if (node._parent !== null)
-            throw new Error('GraphNode is already parented');
-
-        // #if _DEBUG
-        this._debugInsertChild(node);
-        // #endif
-
+        this._prepareInsertChild(node);
```

Comment:
This function now does `node._parent.removeChild(node);` where it didn't before.

Might that change behaviour?

(Same for `insertChild()`).

Reply:
That's what is needed to continue with a valid graph. Better solution than throw.